### PR TITLE
[FIX] UNO 247 - Tooltips of 'Outcome Declarations' in Item properties starts with a letter of lowercase

### PR DESCRIPTION
**Pull request by habemuscode**
**Related to**

https://oat-sa.atlassian.net/browse/UNO-247

**Steps to reproduce**

- Open an item in Authoring mode
- Add any interaction to the canvas
- Click 'Response' in interaction
- Hover the pointer over the question mark icon next to 'Outcome Declarations' section in 'Item Properties to call for a tooltip

**Actual result**

Tooltip starts with lower case letter: 'here you can provide scoring aspects for manual scoring/marking, e.g. CONTENT, GRAMMAR, SPELLING' (please, see the attached screenshot outcome_declaration.png).

image

**Expected result**

Tooltip starts with capital letter: 'Here you can provide scoring aspects for manual scoring/marking, e.g. CONTENT, GRAMMAR, SPELLING'.

### CODE REVIEW

**Comment by habemuscode** [comment]
How it looks like now :)

image

**Comment by habemuscode** [comment]
Instead of just change the text written in the code, I give support to allow translations in the future

File: locales/fr-FR/messages.po [hunk 1]
```diff
@@ -599,6 +599,9 @@ msgstr "Garder pour la ressource \"%s\"..."
 msgid "Height"
 msgstr "Hauteur"
 
+msgid "Here you can provide scoring aspects for manual scoring/marking, e.g. CONTENT, GRAMMAR, SPELLING"
+msgstr ""
```
Comment (andrey-ponamarev) [inline locales/fr-FR/messages.po:603]: It's better to show untranslated message then nothing 

```suggestion
msgid "Here you can provide scoring aspects for manual scoring/marking, e.g. CONTENT, GRAMMAR, SPELLING"
msgstr "Here you can provide scoring aspects for manual scoring/marking, e.g. CONTENT, GRAMMAR, SPELLING"
```

**Review by andrey-ponamarev — APPROVED** [review]
I believe the translation files should be generated automatically during the release

- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [x] New code is not subject to concurrency issues (if applicable)
- [ ] The feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request's target is not `master`

**Review by bziondik — APPROVED** [review]
Next time no need to update `po` files if you don't have translations.
- [ ]  New code is covered by tests (if applicable)
- [ ]  Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x]  New code is respecting code style rules
- [x]  New code is respecting best practices
- [ ]  New code is not subject to concurrency issues (if applicable)
- [x]  Feature is working correctly on my local machine (if applicable)
- [x]  Acceptance criteria are respected
- [x]  Pull request title and description are meaningful

**Pull request merged**